### DISCUSSION TITLE
Closed Source and Open Source Software

### DISCUSSION THREAD

**Discussion by nnhimichelle** [albertkun/21S-ASIAAM-191A#104] [Past Thinking Caps]
Despite the fact that closed source software is provides more usability and can provide more detailed-services, it is also less accessible in some ways and exclusive. With closed software, it can be more difficult to change code to suit your needs, meaning that there could be even more people in which the code has not suit their needs, resulting more people becoming more excluded. Closed source software services also come at a cost in which not everyone can pay for, excluding marginalized communities and people that struggle to live comfortably. 

On the contrary, open source software has loyal online users that are engaging and can provide services with just a click of a forum. However, the problem with this is that when it comes to people's narratives and what is being shared online, people can recycle words or twist them to harm and spread misinformation of communities. In addition, code can easily be recycled and open source software can also put the safety of users at risk with the hacking of these tools to find our information that is personal to user.

**Comment by KenNL42** [comment]
I agree with how closed sourced can be less flexible than open sourced. Closed source generally gear toward a certain audience, while open source offers a wide range of data for people to used. But closed source can be costly, like how Google offers data and API to users per pay if they exceed certain usage limit. In the end, developers have to weigh which source should be used by considering data usability, cost, and data reliability. If developers can find a way to filter data and code from open source software, then open source can be a very powerful tool to use for a project.
